decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb

```diff
@@ -88,7 +88,7 @@ extra_admin_link(
     <% end %>
   </section>
 
-  <% if proposal_has_costs? && current_settings.answers_with_costs? %>
```

Comment:
As far as I see, this helper is only used here, so we can remove the definition too 

```
$ rg proposal_has_costs
decidim-proposals/app/helpers/decidim/proposals/proposals_helper.rb
14:      def proposal_has_costs?
```

Reply:
Oh wait, I realized that we should actually leave this, sorry that I didn't realized before! 

Without this check we would be showing empty costs always. Can you go back with this change (the removal of the helper and the conditional)?